laborator/content/operatii-memorie-gdb/7-pointers/pointers.c

```diff
@@ -16,6 +16,11 @@ int main() {
 
 	char *dest_str = malloc(sizeof(src));
 	char *dest_mem = malloc(sizeof(src));
+	
+	if (!dest_str || !dest_mem) {
```

Comment:
```suggestion
	if (!dest_str || !dest_mem)
	{
```

Reply:
The Linux kernel coding style mentions that brackets should be placed on the next line only for function definitions. Otherwise they are placed on the same line as the instruction that starts the block (`if (cond) {`, `} else {`, `while (cond) {`, etc.). https://www.kernel.org/doc/html/v6.0/process/coding-style.html#placing-braces-and-spaces

Reply:
Oh right. MB, sorry.